mungegithub/mungers/mungerutil/util_test.go

```diff
@@ -111,3 +111,21 @@ func TestIssueUsersAll(t *testing.T) {
 		t.Errorf("AllUsers (%s) doesn't match expected list: %s", users.AllUsers().List(), expected)
 	}
 }
+
+func TestFetchWebFileContents(t *testing.T) {
+	goodurl := "https://storage.googleapis.com/k8s-gubernator/triage/failure_data.json"
```

Comment:
In general, don't write tests that require external network connectivity.

You can use net/http/httptest if you *really* need to test this function. Otherwise I wouldn't bother-- it's pretty trivial.